src/scalems/radical/raptor/__init__.py

```diff
@@ -509,8 +510,9 @@ def launch(cls, manager: ScaleMSMaster, task: TaskDictionary):
                 kwargs={"work_item": work_item},
             )
         )
-        scalems_task = manager.submit_tasks(scalems_task_description)
-        logger.debug(f"Submitted {str(scalems_task)} in support of {str(task)}.")
+        # Note: There is no Task object returned from Master.submit_tasks()
```

Comment:
That is correct - Task objects are currently only available on the client side, the agent side handles task dicts - which the master will receive through the `request_cb`.

Reply:
```suggestion
        # Note: There is no `Task` object returned from `Master.submit_tasks()`
        # `Task` objects are currently only available on the client side;
        # the agent side handles task dicts---which the master will receive through the *request_cb()*.
```

Reply:
To what extent is the task dict schema specified? It isn't exactly the same as `Task.as_dict()`, is it? (There are a few additional implementation details, I think?)